app/views/participant_detail.py

```
@@ -280,4 +269,20 @@ def participant_detail_view():
         "past_year_participation": past_year_participation,
     }
 
-    return render_template("others/participant_detail.html", **context)
+    return render_template("participant_detail/participant_detail.html", **context)
+
+
+# MARK: リダイレクト
+def participant_detail_deprecated_view():
+    """
+    参加者詳細情報を表示する（deprecated）。
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)